crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -788,8 +810,16 @@ async fn process_smartrest(
                 }
             }
         }
-        // Ignore all operations for child devices as not yet supported
-        _ => Ok(vec![]),
+        _ => {
+            match get_smartrest_template_id(payload).as_str() {
+                "106" => register_child_device_supported_operations(config_dir, payload, children),
+                // Ignore any other child device incoming request as not yet supported
+                _ => {
+                    info!("Ignored. Message not yet supported: {payload}");
```

Comment:
```suggestion
                    debug!("Ignored. Message not yet supported: {payload}");
```
This might be better as a debug message otherwise I see the mapper logging this for every operation request sent to any child device.

Reply:
`debug!` or `info!`? I proposed `info!` because it's not convenient when you have to change the log level to discover that the messages sent to the device are ignored.  But I agree that if this is a normal case to have these messages ignored by the mapper and processed by a plugin, then `debug!` is more appropriate.

Reply:
When we extend the mapper in future where it performs all the message mapping for all operations(including config mgmt,log mgmt etc), even for child devices, `info` or even `warn` logging would be the right thing to do as we would want to know when the device receives a message that the mapper can not handle. But today, since the mapper only maps software management operations for the parent device only, I feel that this log will be triggered most of the time.